cypress/integration/page.spec.js

```diff
@@ -556,26 +585,26 @@
   describe('Posts List', () => {
     beforeEach(() => {
       page.mockUsers();
-      page.mockUser1Posts()
+      page.mockUser1Posts();
       cy.visit('/');
 
       userSelector.select(0);
       page.waitForRequest('@user1PostsRequest');
     });
 
     it('should not have posts with Close buttons', () => {
-      cy.contains('[data-cy="PageButton"]', 'Close').should('not.exist')
+      cy.contains('[data-cy="PageButton"]', 'Close').should('not.exist');
```

Comment:
The selector '[data-cy="PageButton"]' seems incorrect as it should likely be '[data-cy="PostButton"]'. Verify the correct data-cy attribute for the post buttons.